build.fsx

```diff
@@ -132,9 +137,59 @@ Target "RunTests" <| fun _ ->
     let xunitToolPath = findToolInSubPath "xunit.console.exe" "src/packages/xunit.runner.console*/tools"
     printfn "Using XUnit runner: %s" xunitToolPath
     xUnit2
-        (fun p -> { p with OutputDir = testOutput; ToolPath = xunitToolPath })
+        (fun p -> { p with HtmlOutputPath = Some(testOutput @@ "xunit.html") })
         xunitTestAssemblies
 
+Target "StartDbContainer" <| fun _ ->
+    let dockerImage = getBuildParamOrDefault "dockerImage" @"microsoft/mssql-server-windows-express"
+    logfn "Starting SQL Express Docker container using image: %s" dockerImage
+    let posh = PowerShell.Create().AddScript(sprintf @"./docker_sql_express.ps1 -dockerImage %s" dockerImage)
```

Comment:
While I was looking at different routes other than Powershell for this, I discovered `Fake.ProcessHelper`.  I'd like to explore this route because if we were to transition from Powershell to using `Fake.ProcessHelper`, we could call `ExecProcess (fun info -> info.FileName <- "docker.exe", info.Arguments <- "...")`.  This might be better from an x-plat perspective.  There is also the opportunity to switch on whether the build `isMono` using `Fake.EnvironmentHelper` as an option if we want to use a bash version of the Powershell script.